build/NuSpecs/UmbracoCms.Core.nuspec

```diff
@@ -23,40 +23,23 @@
       <dependency id="AutoMapper" version="[7.0.1,7.999999)" />
       <dependency id="ClientDependency" version="[1.9.6,1.999999)" />
       <dependency id="ClientDependency-Mvc5" version="[1.8.0,1.999999)" />
-      <dependency id="CSharpTest.Net.Collections" version="[14.906.1403.1082,14.999999)" />
-      <dependency id="Examine" version="[1.0.0-beta025,1.999999)" />
       <dependency id="HtmlAgilityPack" version="[1.7.2,1.999999)" />
       <dependency id="ImageProcessor" version="[2.6.1.19,2.999999)" />
-      <dependency id="ImageProcessor.Web" version="[4.9.2.19,4.999999)" />
-      <dependency id="ImageProcessor.Web.Config" version="[2.4.1.19,2.999999)" />
       <dependency id="LightInject" version="[5.1.2,5.999999)" />
       <dependency id="LightInject.Annotation" version="[1.1.0,1.999999)" />
       <dependency id="LightInject.Mvc" version="[2.0.0,2.999999)" />
       <dependency id="LightInject.WebApi" version="[2.0.0,2.999999)" />
       <dependency id="log4net" version="[2.0.8,2.999999)" />
-      <dependency id="Log4Net.Async" version="[2.0.4,2.999999)" />
-      <dependency id="Lucene.Net.Contrib" version="[3.0.3,3.999999)" />
-      <dependency id="Markdown" version="[2.2.1,2.999999)" />
       <dependency id="Microsoft.AspNet.Identity.Owin" version="[2.2.1,2.999999)" />
-      <dependency id="Microsoft.AspNet.Mvc" version="[5.2.6,5.999999)" />
-      <dependency id="Microsoft.AspNet.SignalR.Core" version="[2.2.3,2.999999)" />
       <dependency id="Microsoft.AspNet.WebApi" version="[5.2.6,5.999999)" />
-      <dependency id="Microsoft.CodeAnalysis.CSharp" version="[2.8.0,2.999999)" />
-      <dependency id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="[1.0.8,1.999999)" />
-      <dependency id="Microsoft.Net.Compilers" version="[2.8.0,2.999999)" />
       <dependency id="Microsoft.Owin.Host.SystemWeb" version="[4.0.0,4.999999)" />
       <dependency id="Microsoft.Owin.Security.Cookies" version="[4.0.0,4.999999)" />
       <dependency id="Microsoft.Owin.Security.OAuth" version="[4.0.0,4.999999)" />
       <dependency id="Microsoft.Web.Xdt" version="[2.1.2,2.999999)" />
       <dependency id="MiniProfiler" version="[3.2.0.157,3.999999)" />
       <dependency id="MySql.Data" version="[6.10.7,6.999999)" />
       <dependency id="Newtonsoft.Json" version="[11.0.2,11.999999)" />
-      <dependency id="NPoco" version="[3.9.3,3.999999)" />
       <dependency id="Semver" version="[2.0.4,2.999999)" />
-      <dependency id="System.Reflection.Primitives" version="[4.3.0,4.999999)" />
```

Comment:
Not sure about these 4, any particular reason these were removed as dependencies for the Core nuspec?